src/tests/ChainIds.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.22;
+
+library ChainIds {
+    /*//////////////////////////////////////////////////////////////////////////
+                                      MAINNETS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    uint256 public constant MAINNET = 1;
+    uint256 public constant ARBITRUM = 42_161;
+    uint256 public constant AVALANCHE = 43_114;
+    uint256 public constant BASE = 8453;
+    uint256 public constant BERACHAIN = 80_094;
+    uint256 public constant BLAST = 81_457;
+    uint256 public constant BSC = 56;
+    uint256 public constant GNOSIS = 100;
+    uint256 public constant LIGHTLINK = 1890;
+    uint256 public constant LINEA = 59_144;
+    uint256 public constant MODE = 34_443;
+    uint256 public constant MORPH = 2818;
+    uint256 public constant OPTIMISM = 10;
+    uint256 public constant POLYGON = 137;
+    uint256 public constant CHILIZ = 88_888;
+    uint256 public constant COREDAO = 1116;
+    uint256 public constant FORM = 478;
+    uint256 public constant SCROLL = 534_352;
+    uint256 public constant SUPERSEED = 5330;
+    uint256 public constant TAIKO = 167_000;
+    uint256 public constant XDC = 50;
```

Comment:
1. Can we order them alphabetically, please?
2. can we rename `MAINNET` to `ETHEREUM`
3. Where's zkSync?